dex/testing/dcr/vspdtemplates/header.html

```diff
@@ -0,0 +1,76 @@
+{{define "header"}}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <title>Decred VSP - {{ .WebApiCfg.Designation }}</title>
+
+        <link rel="stylesheet" href="/public/css/vendor/bootstrap-4.5.0.min.css" />
+        <link rel="stylesheet" href="/public/css/vspd.css?v={{ .WebApiCfg.VspdVersion }}" />
+        <!-- fonts.css should be last to ensure dcr fonts take precedence. -->
+        <link rel="stylesheet" href="/public/css/fonts.css?v={{ .WebApiCfg.VspdVersion }}" />
+
+        <!--  Custom favicon  -->
+        <!-- Apple PWA -->
+        <link rel="apple-touch-icon" sizes="57x57"   href="/public/images/favicon/apple-touch-icon-57x57.png?v=gT6Mc">
```

Comment:
Looks like we don't intend for the web interface for vspd to actually work?  We're not checking in the stuff in /public (and I don't want to), so are these html templates just in the repository so vspd doesn't choke on startup?  If so, we can just make stubs for all these html files that define mostly-empty templates that execute without error.

Reply:
Or should we have git clone of vspd be part of create-vpsd.sh since vspd doesn't (yet) embed these resources.

Reply:
> Or should we have git clone of vspd be part of create-vpsd.sh since vspd doesn't (yet) embed these resources.

This is the better idea. Will do.